R/page.R

```diff
@@ -27,7 +27,8 @@ page <- function(..., title = NULL, theme = bs_theme(), lang = NULL) {
       theme = theme,
       lang = lang,
       # Components require Bootstrap 5+
-      if (isTRUE(theme_version(theme) >= 5)) component_dependencies()
+      if (isTRUE(theme_version(theme) >= 5)) component_dependencies(),
+      if (is_installed("shiny", "1.8.1.9001")) shiny::useBusyIndicators()
```

Comment:
Maybe we should rename `is_installed(version=)` to `gt_eq` so that we can write this

```r
if (is_installed("shiny", gt_eq = "1.8.1.9000"))
```

Otherwise you have to go read the source of `is_installed()` (like I just did) or remember that `version` means `>=`.